### ISSUE TITLE
`<input type=text placeholder="some text">`で「アクセシブルな名前が必要です」と返される

### ISSUE THREAD

**Issue by momdo** [markuplint/markuplint#1477]
- Markuplint version: `4.1.0`
- Parser lang: HTML
- Node.js version: `20.11.0`
- OS: Windows

## Describe the bug

`<input type=text placeholder="some text">`で次のメッセージが出力されます。

> アクセシブルな名前が必要です

しかし、[HTML-AAM仕様§4.1.1](https://www.w3.org/TR/html-aam-1.0/#input-type-text-input-type-password-input-type-number-input-type-search-input-type-tel-input-type-email-input-type-url-and-textarea-element-accessible-name-computation)によれば、`placeholder`は名前として計算されますから、このエラーメッセージは適切ではないです。

## Code Example

```html
<input type="text" placeholder="require-accessible-name">
```

## Steps To Reproduce

1. `<input type=text placeholder="some text">`を記述します

## The current behavior

アクセシブルな名前が必要です

と出力されます。

## The expected behavior

- エラーとして報告しないようにします。もしくは、
- エラーとして報告するのであれば「`placeholder`属性しかない」ことをメッセージで指摘するようにします。

**Comment by YusukeHirao** [comment]
Fixed #1501
